app/views/admins/dashboard.mobile.haml

```diff
@@ -0,0 +1,4 @@
+- content_for :head do
+  = stylesheet_link_tag :admin
```

Comment:
Do we really need linking this stylesheet here? In the current version it isn't included in desktop version and no elements in dashboard (both versions) really seem to rely on the styles defined in `admin.css`.